models/Tool/UUID/Dao.php

```diff
@@ -64,4 +77,16 @@ public function getByUuid($uuid)
 
         return $model;
     }
+
+    /**
+     * @param string $uuid
+     *
+     * @return bool
+     */
+    public function exists($uuid)
+    {
+        $result = $this->db->fetchOne('SELECT EXISTS(SELECT uuid FROM ' . self::TABLE_NAME . ' where uuid = ?)', [$uuid]);
```

Comment:
```suggestion
        $result = $this->db->fetchOne('SELECT uuid FROM ' . self::TABLE_NAME . ' where uuid = ?', [$uuid]);
```
I think we can omit the `EXISTS()` here, as it makes no difference, right?